src/doc/reference/glossary/cpu.md

```diff
@@ -0,0 +1,7 @@
+---
+title: CPU
+tags: cpu
+---
+"Central Processing Unit"; we use this
```

Comment:
Formatting-wise, I'd suggest:

Central Processing Unit.  We use this

(In olther words, no quotes, and use a period instead of a semicolon.)